packages/gatsby-cli/src/init-starter.js

```diff
@@ -177,9 +224,21 @@ type InitOptions = {
  * Main function that clones or copies the starter.
  */
 module.exports = async (starter: string, options: InitOptions = {}) => {
-  const rootPath = options.rootPath || process.cwd()
+  const { starterPath, rootPath, selectedOtherStarter } = await getPaths(
+    starter,
+    options.rootPath
+  )
 
   const urlObject = url.parse(rootPath)
+
+  if (selectedOtherStarter) {
+    report.log(
```

Comment:
report.info might be a better fit
```suggestion
    report.info(
```